src/main/scala/treadle/executable/ExecutionEngine.scala

```diff
@@ -153,6 +157,43 @@ class ExecutionEngine(
     }
   }
 
+  def randomize(): Unit = {
+    val rand = new Random
+
+//    val symbolsToDo: Seq[Symbol] = {
```

Comment:
you should remove this code or uncomment it

Reply:
removed code